model-optimizer/extensions/back/ReverseInputChannels.py

```diff
@@ -314,19 +313,21 @@ def lift_up_through(node: Node, reverse_channels: Node):
                     |
                  next_op
 
-        returns boolean value whatever we should continue propagating current ReverseChannels operation up or not
+        returns two objects:
+        first - boolean value whatever we should continue propagating current ReverseChannels operation up or not
+        second - list of ReverseChannels operations that were produced while propagating reverse_channels up
         """
         if node.is_in_port_connected(0):
             node_input_port_0 = node.in_port(0)
-            reverse_channels_out_npde = reverse_channels.out_port(0).get_connection().get_destination().node
+            reverse_channels_out_node = reverse_channels.out_port(0).get_connection().get_destination().node
```

Comment:
what if `ReverseChannels` has several destinations? Now this functions seems to have a limitation.